assets/component-price.css

```diff
@@ -64,10 +64,15 @@
 }
```

Comment:
Should we be showing currency here too? https://screenshot.click/18-00-xuvlt-bvr1n.png

Reply:
Yes, we should.

Reply:
Seems like B2B prices do not respect the theme's global currency code setting. Is this ok?

![Screenshot 2023-07-19 at 10 34 26 AM](https://github.com/Shopify/dawn/assets/3710548/31dcc767-2906-4aaf-b3b0-e770c764c44f)

![Screenshot 2023-07-19 at 10 34 37 AM](https://github.com/Shopify/dawn/assets/3710548/973b5205-1180-40c0-8283-71838ad9ccd1)

Reply:
I think we had decided on having it always show on the PDP (list and price per item) and the quick order list never show. But unsure if we should follow it cc @nicklepine @edmund-teh

Reply:
Yes let's show currency code on price block of product info section. this seems like a regression given it's showing for me on my test store using the zip you created for me @KaichenWang on July 18. 

https://screenshot.click/19-00-b84dd-f3jfs.mp4

Reply:
Note: we have a follow up ticket to make sure this respects the global theme setting, but for now lets just show it. https://github.com/Shopify/dawn/issues/2854

Reply:
Okay, as I understand everything works okay here for now. 👍